README.md

```diff
@@ -80,8 +80,8 @@ can also specify a different filename on the command line such as:
 ```console
 $ npm start my-org-file.toml
 
-> github_membership@1.0.0 start
-> node index.js
+> orglinter@1.0.0 start
+> node ./src/bin/cli.js
```

Comment:
I would suggest separating `Script Execution` into two sections: `Usage` and `Local Development`

``` md
## Usage
Orglinter is designed to be installed with npm or yarn:

**Globally**
npm install -g orglinter
orglinter my-org-file.toml

**Within your package**
npm install orglinter
npx orglinter my-org-file.toml

## Local Development

If you want to make contributions to `orglinter` you can using `git` and `npm`:

git clone git@github.com:godaddy/orglinter
cd orglinter
npm install
npm start my-org-file.toml
```